### PR TITLE
Add a conversion script for a RoBERTa checkpoints to HDF5

### PR DESCRIPTION
**Pull request by danieldk**
- Add a conversion script for converting a PyTorch RoBERTa checkpoint
  to HDF5.
- Adjust the BERT conversion script to produce HDF5 without the weird
  Tensorflow/BERT nomenclature: kernel/gamma/beta. Instead just use
  weight/weight/bias as RoBERTa does.
- Change the LoadFromHDF5 implementations to use the weight/bias
  naming.

This does not break compatibility with existing models, since we
already use weight/bias naming internally.

### CODE REVIEW

**Comment by danieldk** [comment]
Updated this PR after testing: the RoBERTa embedding layers did have non-matching names, which is fixed now.

**Review by twuebi — APPROVED** [review]
looks fine, good to go if the test run is positive

**Comment by danieldk** [comment]
Thanks for the review! The model did not seem to get far accuracy-wise (at least on dependencies). I saw now that the RoBERTa models need an offset in the position indices. Doing another run to see if that was the problem. I'll merge this once training seems to lead somewhere.

**Comment by twuebi** [comment]
Will look at the tokenization pr later

**Comment by danieldk** [comment]
One bug down: it turns out that all the non-embedding matrices are transposed compared to BERT. I have updated this PR. Since I cross-check again against HF transformers I know the conversion to be correct now, so I will merge this once CI is done.
